### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by NickyArias**
Added samples for CLI/Web and ReactNative/ReactNative0.64.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,9 @@ This repo contains samples that demonstrate configuring PreEmptive Protection JS
 - [Electron](Electron): demonstrates how you can use the JSDefender Webpack plugin to protect Electron projects.
 - [Angular](Angular): demonstrates how you can use the JSDefender Webpack plugin to protect Angular projects.
 - [Ionic](Ionic): demonstrates how you can use the JSDefender Webpack plugin to protect different Ionic projects.
+- [CLI/Web](CLI/Web): demonstrates how you can use the JSDefender Webpack plugin to protect Vanilla JavaScript projects.
```
Comment (dominikfoldi) [inline README.md:16]: @NickyArias use this instead (feel free to fix grammatical or wording issues):
[CLI](CLI): demonstrates how you can use the JSDefender CLI to protect JavaScript projects with multiple input files e.g. a Vanilla JavaScript Web app without any bundler.

File: README.md [hunk 2]
```diff
@@ -13,7 +13,9 @@ This repo contains samples that demonstrate configuring PreEmptive Protection JS
...
+- [ReactNative/ReacNative0.64](ReactNative/ReacNative0.64): demonstrates how you can use the JSDefender Webpack plugin to protect React Native projects.
```
Comment (dominikfoldi) [inline README.md:17]: @NickyArias use this instead (feel free to fix grammatical or wording issues):
[ReactNative](ReactNative): demonstrates how you can use the JSDefender Metro plugin to protect React Native projects.

Comment (NickyArias) [reply README.md]: @dominikfoldi changes were applied.

File: README.md (hunk 1 above)
Comment (dominikfoldi) [reply README.md]: @NickyArias you forgot to update the beginning: `[CLI](CLI)`

File: README.md (hunk 2 above)
Comment (dominikfoldi) [reply README.md]: @NickyArias you forgot to update the beginning: `[ReactNative](ReactNative)`

**Pull request merged**